web/app.js

```diff
@@ -2000,7 +2000,7 @@ function webViewerPageChanging(e) {
 var zoomDisabled = false, zoomDisabledTimeout;
 function handleMouseWheel(evt) {
   var pdfViewer = PDFViewerApplication.pdfViewer;
-  if (pdfViewer.isInPresentationMode) {
+  if (pdfViewer && pdfViewer.isInPresentationMode) {
```

Comment:
If the `pdfViewer` object is not defined, then we will also be in trouble in the other parts of this function. Therefore, wouldn't it be better to change this line to `if (!pdfViewer || pdfViewer.isInPresentationMode) {`? That way we get out if `pdfViewer` is not defined (to avoid further problems) and also when we're in presentation mode.